### PR TITLE
feat/migrate airtable source connector

### PR DESCRIPTION
**Pull request by rbiseck3**
### Description
Handles issue https://github.com/Unstructured-IO/unstructured-ingest/issues/97

### CODE REVIEW

File: unstructured_ingest/v2/examples/airtable.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    logger.info(f"Writing all content in: {work_dir.resolve()}")
+    Pipeline.from_configs(
+        context=ProcessorConfig(work_dir=str(work_dir.resolve()), verbose=True),
+        indexer_config=AirtableIndexerConfig(
+            list_of_paths=["app5YQxSfp220fWtm", "appJ43QmP8I17zu88"]
+        ),
+        downloader_config=AirtableDownloaderConfig(download_dir=download_path),
+        source_connection_config=AirtableConnectionConfig(
+            access_config=AirtableAccessConfig(
+                personal_access_token=os.getenv("AIRTABLE_PERSONAL_ACCESS_TOKEN")
```
Comment (hubert-rutkowski85) [inline unstructured_ingest/v2/examples/airtable.py:37]: I understand for running this test locally, developer needs to register to Airtable, put there files from `test_e2e/expected-structured-output/airtable-diff/app5YQxSfp220fWtm` and set his PAT as this env. Is that right? 

Btw, I also see Airtable mentioned in e2e.yaml in GH action. So for that scenario, there is the code in `test_e2e/env_setup/airtable/create_scale_test_components.py` , and it's going to be run also for this v2 Airtable?

**Review by hubert-rutkowski85 — APPROVED** [review]
LGTM, just two questions

File: unstructured_ingest/v2/examples/airtable.py (hunk 1 above)
Comment (rbiseck3) [reply unstructured_ingest/v2/examples/airtable.py:37]: Now that v2 is exposed, it's what will be used by the ingest tests.

File: unstructured_ingest/v2/processes/connectors/airtable.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            elif len(components) == 2:
+                airtable_meta.append(
+                    AirtableTableMeta(base_id=components[0], table_id=components[1])
+                )
+            else:
+                airtable_meta.append(
+                    AirtableTableMeta(
+                        base_id=components[0], table_id=components[1], view_id=components[2]
+                    )
+                )
```
[the remark this answers is not in the document]
Comment (rbiseck3) [reply unstructured_ingest/v2/processes/connectors/airtable.py]: Added better validation.

**Pull request merged**